core/src/main/scala/reftree/render/Renderer.scala

```diff
@@ -20,7 +20,8 @@ import reftree.svg.SvgGraphAnimation
  *
  *   val renderer = Renderer(
  *     renderingOptions = RenderingOptions(density = 75),
- *     directory = Paths.get("images", "usage")
+ *     directory = Paths.get("images", "usage"),
+ *     format = "jpg"
```

Comment:
As above, could you include two examples, without specifying `format` and with `format = "gif"`?